spec/font_spec.rb

```diff
@@ -95,7 +95,7 @@
         input = input_lines.join %(\n\n)
         pdf = to_pdf input, attribute_overrides: { 'pdf-theme' => 'default-with-fallback-font' }, analyze: true
         (expect pdf.lines).to eql input_lines
-      end).to log_message severity: :WARN, message: %(Could not locate the character `\u20bf' in the following fonts: Noto Serif, M+ 1p Fallback, Noto Emoji), using_log_level: :INFO
+      end).to log_message severity: :WARN, message: %(Could not locate the character `\u20bf' (\\u20BF) in the following fonts: Noto Serif, M+ 1p Fallback, Noto Emoji), using_log_level: :INFO
```

Comment:
When using unpack, this will change to lowercase:

```suggestion
      end).to log_message severity: :WARN, message: %(Could not locate the character `\u20bf' (\\u20bf) in the following fonts: Noto Serif, M+ 1p Fallback, Noto Emoji), using_log_level: :INFO
```